util/import/BUILD.bazel

```diff
@@ -1,11 +1,11 @@
 load("//rust:defs.bzl", "rust_library", "rust_proc_macro", "rust_test")
 
 # buildifier: disable=bzl-visibility
-load("//rust/private:transitions.bzl", "with_import_macro_bootstrapping_mode")
+load("//rust/private:transitions.bzl", "alias_with_import_macro_bootstrapping_mode")
 
-with_import_macro_bootstrapping_mode(
+alias_with_import_macro_bootstrapping_mode(
     name = "import_macro",
-    target = "import_macro_impl",
+    deps = ["import_macro_impl"],
```

Comment:
Could this not be `default = [Label("@rules_rust//uril/import:import_macro_impl")]` on the attribute?

Reply:
It could, but I think I prefer the current setup - scanning through the BUILD file gives you the idea what's going on. With the `default` trick you have to go to the bzl file and back to follow the breadcrumbs... Are there advantages that I don't see?

Reply:
I think the default is probably a better way to handle this in this case. If it's logically a default, that's how it should be modeled.

Reply:
If `alias_with_import_macro_bootstrapping_mode` is expected to be used anywhere else, I'd say `default` should be used so other users don't have to think about it. Not sure if it's directly relevant to this change but it'd be nice to see all the doc fields filled out on the rule to explain what it's purpose was. Can you elaborate on it a bit here though?

Reply:
I don't see `alias_with_import_macro_bootstrapping_mode` as being logically tied to the import_macro. It's a rule that changes the configuration for its `deps`/`target`. And I really value the readability and explicitness of the BUILD file. To be honest I haven't seen a non-implicit label attribute with a default yet, I think they are rare and surprising.

I think the optimal solution would have been to create a `rust_binary`-like rule that does everything the standard `rust_binary` rule does plus it does the transition. That is sadly a bit too complicated and full of boilerplate right now.

Reply:
I don't care too much about it being a default. Had I written this, I would generally opt to keep private/implementation-detail rules as concise as possible by implementing the majority of the funcitonality in the `.bzl` file to have a single source of truth and only ever specifying `my_rule(name = "my_rule")` in the BUILD file. I find this makes things far more readable and easier to debug.